java/bulk-import/bulk-import-starter/src/main/java/sleeper/bulkimport/starter/executor/EmrExecutor.java

```diff
@@ -74,8 +78,22 @@ public class EmrExecutor extends AbstractEmrExecutor {
     public EmrExecutor(AmazonElasticMapReduce emrClient,
                        InstanceProperties instanceProperties,
                        TablePropertiesProvider tablePropertiesProvider,
-                       StateStoreProvider stateStoreProvider, AmazonS3 amazonS3) {
-        super(instanceProperties, tablePropertiesProvider, stateStoreProvider, amazonS3);
+                       StateStoreProvider stateStoreProvider,
+                       IngestJobStatusStore ingestJobStatusStore,
+                       AmazonS3 s3Client) {
+        this(emrClient, instanceProperties, tablePropertiesProvider, stateStoreProvider, ingestJobStatusStore,
+                s3Client, UUID.randomUUID().toString(), Instant::now);
```

Comment:
Could we make EmrExecutorTest use ExecutorFactory to build the executor in a helper, so that we don't have to default these twice?

That would also make it easier to factor out the abstract class in case we want to avoid passing these variables into every executor type.